harness/determined/deploy/aws/templates/fsx.yaml

```diff
@@ -195,6 +195,11 @@ Parameters:
     Type: String
     Description: preexisting FSx ID to mount
     Default: ""
+  
+  WebhooksBaseUrl:
```

Comment:
i imagine there has to be a way to autopopulate this field for the average deployment (that just uses the public dns record assigned by aws, and doesn't also have a record for, e.g. `latest-master.determined.ai`

Reply:
@stoksc 

If what you said above cannot be implemented, is this the file where I need to define the default URL to `http://latest-master.determined.ai:8080/`?

Reply:
Oh yea that makes sense. Thanks!